### PR TITLE
Testing and general maintenace

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR moves to using Jest, adds prettier, updates some dependencies and also removes [UglifyES in favor of Terser](https://twitter.com/sebmarkbage/status/1060053812075872256?s=20).

Please test using the app or sandbox to see everything is running smooth.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,51 +39,65 @@
...
+    "*.{js,jsx}": [
+      "prettier --config .prettierrc --write",
+      "eslint -c .eslintrc.json --ext .jsx,.js --fix",
+      "git add"
+    ]
+  },
   "devDependencies": {
-    "@deck.gl/core": "^7.1.10",
-    "@deck.gl/mapbox": "^7.1.10",
+    "@deck.gl/core": "7.1.10",
```
Comment (edbrett) [inline package.json]: Any reason to fix the versions here and not in other cases when we have a yarn.lock?

File: rollup.config.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import babel from 'rollup-plugin-babel';
 import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
-import { uglify } from 'rollup-plugin-uglify';
-import { minify } from 'uglify-es';
+import { terser } from 'rollup-plugin-terser';
 import pkg from './package.json';
 
+const terserOptions = {
+  safari10: true
```
Comment (edbrett) [inline rollup.config.js]: I hate safari 10.

File: rollup.config.js [hunk 3]
```diff
@@ -77,7 +80,7 @@ export default [
       globals,
     },
     external,
-    plugins: [babel(babelOptions()), resolve(), commonjs(), uglify({}, minify)]
+    plugins: [babel(babelOptions()), resolve(), commonjs(), terser(terserOptions)]
```
Comment (edbrett) [inline rollup.config.js]: lovely.

File: package.json (hunk 1 above)
Comment (sorodrigo) [reply package.json]: cause @deck.gl is not following semver

**Pull request merged**